src/components/StatsTask/TokenInput/TokenInput.tsx

```diff
@@ -0,0 +1,185 @@
+import React, { useRef, useState } from 'react'
```

Comment:
you could use one of inputs that are already in the project instead of creating a new one

Reply:
Once again: you can use one of inputs that we already have in project instead of creating a new one - I see that this is copypasted DepositAmountInput - why don't use this?